app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -4,20 +4,38 @@
 import com.codesoom.assignment.domain.Product;
 import com.codesoom.assignment.domain.ProductRepository;
 import com.codesoom.assignment.dto.ProductData;
+import com.github.dozermapper.core.Mapper;
+
 import org.springframework.stereotype.Service;
 
 import javax.transaction.Transactional;
 import java.util.List;
 
+/**
+ * Product 리소스
+ * 수행할 작업을 정의하고, Domain layer에 수행을 위임한다.
+ *
+ * @see Product
+ * @see ProductRepository
+ */
```

Comment:
이 정도면 충분하겠죠. `Domain layer에 수행을 위임한다.`처럼 설명을 적으면 마치 자신이 만화 속 인물인 것을 알고 있는 만화 주인공에게 자기 소개를 부탁한 것 같지 않나요?

```suggestion
/**
 * Product에 대한 조회, 입력, 수정 등을 담당합니다.
 */
```

한편 `@see ProductRepository`는 캡슐화를 깹니다. 클래스 javadoc은 클래스 바깥으로 공개되는 정보라고 생각하세요. 클래스 바깥으로 공개되는 정보에서 클래스 안쪽에서 몰래 사용하고 있는 `ProductRepository`를 드러낼 필요는 없습니다.

Reply:
아! 그렇군요! 수정하겠습니다. 감사합니다.